packages/headless/src/api/commerce/commerce-api-client.test.ts

```diff
@@ -46,6 +46,7 @@ describe('commerce api client', () => {
         url: '',
```

Comment:
The current approach works, but it's not consistent with the other engines, where the value of `capture` is set based on whether analytics are enabled or not. Is there a need to have direct control over the capture flag with commerce engine, or could we use a similar pattern?

Reply:
Doh! That's a much better approach. I don't think there's a need for direct control of the `capture` property. I'll make the change.